tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -143,3 +143,32 @@ def test_check_mds_cache_memory_limit(self):
         log.info(
             f"mds_cache_memory_limit is set with a value of {expected_mds_value_in_GB}GB"
         )
+
+    @bugzilla("2012930")
+    @post_ocs_upgrade
+    @pytest.mark.polarion_id("OCS-2739")
+    def test_noobaa_postgres_cm_post_ocs_upgrade(self):
+        """
+        Validate noobaa postgres configmap post OCS upgrade
+
+        """
+        cm_obj = OCP(
+            kind=constants.CONFIGMAP,
+            resource_name=constants.NOOBAA_POSTGRES_CONFIGMAP,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+        )
+        config_data = cm_obj.get().get("data").get("noobaa-postgres.conf")
+        config_data = config_data.split("\n")
+        config_data = config_data[slice(5, len(config_data))]
+        log.info(
+            "Validating that the values configured in noobaa-postgres configmap "
+            "match the ones stored in ocs-ci"
+        )
+        stored_values = constants.NOOBAA_POSTGRES_TUNING_VALUES.split("\n")
+        stored_values.remove("")
+        assert collections.Counter(config_data) == collections.Counter(stored_values), (
+            f"The config set in {constants.NOOBAA_POSTGRES_CONFIGMAP} "
+            f"is different than the expected. Please inform OCS-QE about this discrepancy. "
+            f"The expected values are:\n{stored_values}\n"
+            f"The cluster's existing values are:{config_data}"
+        )
```

Comment:
Have you tried how would a failure look like in the logs?

Reply:
I dont think this will throw error in logs, As those values are editable in the configmap